contrib/go-redis/redis.v7/redis_test.go

```diff
@@ -277,24 +272,23 @@ func TestChildSpan(t *testing.T) {
 	}
 	assert.NotNil(parent)
 	assert.NotNil(child)
-
+	assert.Equal(ctx, client.Context())
```

Comment:
This assertion is not needed. This is the responsibility of the upstream library to test.